lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -237,7 +237,7 @@ def define
 
       outcome :outcome_opposite_sex_marriage_poland_residing_in_uk_or_ceremony_country
 
-      outcome :outcome_os_slovenia
+      outcome :outcome_opposite_sex_marriage_slovenia_residing_in_uk_or_ceremony_country
```

Comment:
Minor: It might be worth explaining why you chose to use `outcome_opposite_sex_marriage_slovenia_residing_in_uk_or_ceremony_country` and not `outcome_opposite_sex_marriage_slovenia_residing_in_uk_or_slovenia` here.